frontend/src/metabase/query_builder/components/DataSelector/data-search/SearchResults.jsx

```diff
@@ -55,7 +55,7 @@ export function SearchResults({
                     result={item}
                     onClick={onSelect}
                     compact
-                    hasDescription={false}
+                    showDescription={false}
```

Comment:
Change the name because `hasDescription` implies the possession of a `description` property rather than saying `please show the description`